File: composeApp/src/androidMain/kotlin/com/github/eylulnc/notia/feature/settings/common/SettingsUiState.kt
package com.github.eylulnc.notia.feature.settings.common

import com.github.eylulnc.notia.ui.theme.ThemeMode
import kotlinx.datetime.LocalTime

data class SettingsUiState(
    val themeMode: ThemeMode,
    val appVersion: String,
    val isResetDialogVisible: Boolean = false,
    val isReminderEnabled: Boolean = false,
    val reminderTime: LocalTime = LocalTime(8, 30)
)

File: iosApp/iosApp/Features/Settings/SettingsUiState.swift
import Foundation

struct SettingsUiState {
    var themeMode: ThemeMode = .system
    var appVersion: String = ""
    var isResetDialogVisible: Bool = false
    var isReminderEnabled: Bool = false
    var reminderHour: Int = 8
    var reminderMinute: Int = 30
    var showNotificationDeniedAlert: Bool = false
}
